### ISSUE TITLE
JAWS not reading <caption> value when role=presentation on table

### ISSUE THREAD

**Issue by allanj-uaag** [FreedomScientific/VFO-standards-support#395]
When using a table to layout a form. the table is marked with role="presentation". the caption text is ignored by JAWS

  
### Expected result

When arrowing through the page the `<h2>` within the caption should be read along with the content.
the valid heading (which is in the DOM) should be read
the heading should show in the Heading list.

### Actual result

none of the above.
heading not read when arrowing through page
heading not read when navigating by heading
heading does not appear in heading list

### Example

https://codepen.io/jimallan/pen/mdVVxGX

#### JAWS version and build number

2020.2004.66
 
#### Operating System and version
Win10

#### Browser and version:
Chrome Version 81.0.4044.138

**Comment by JAWS-test** [comment]
@stevefaulkner I understand the ARIA specification for role=presentation to mean that it should be possible that all text content should still be perceptible, regardless of the table elements used

**Comment by allanj-uaag** [comment]
@stevefaulkner Agree. this was a repair. Original form was  a table with caption and form controls. JAWS read the column information as well as the form info... lots of noise for the user. role=presentation solved that problem.  but the caption didn't read. I thought role=presentation is supposed to ignore the table markup and read the rest of the content . The caption content has content and should be read regardless of the advice.

**Comment by JeremiahRogersVispero** [comment]
Using JAWS 2026.2606.132, this works as described by @allanj-uaag in the Expected result section of the issue content.
